component-generator/templates/README.md

```diff
@@ -15,7 +15,5 @@ Describe your component here and give usage examples.
 ### Basic Usage
 
 ```jsx live
-<componentName>
-  Hello!
-</componentName>
+<componentName>Hello!</componentName>
```

Comment:
Cand we not change this? The way it was before looks more readable to me

Reply:
It seems to me that it is expected that the code that fits on one line will have this format